src/lib.rs

```diff
@@ -118,7 +118,6 @@
 	clippy::cast_possible_wrap,
 	clippy::cast_possible_truncation,
 	clippy::module_name_repetitions,
-	clippy::must_use_candidate,
```

Comment:
I'd rather keep this `allow`ed, it's a pretty noisy lint when trying to iterate.